code/modules/antagonists/heretic/knowledge/starting_lore.dm

```diff
@@ -0,0 +1,128 @@
+	name = "The Living Heart"
+	desc = "Grants you a Living Heart, allowing you to track sacrifice targets. \
+		Should you lose your heart, you can transmute a poppy and a pool of blood \
+		to awaken your heart into a Living Heart. If your heart is cybernetic, \
+		you will additionally require a usable organic heart in the transmutation."
+	required_atoms = list(
+		/obj/effect/decal/cleanable/blood = 1,
+		/obj/item/food/grown/poppy = 1,
+	)
+	cost = 0
+	route = PATH_START
+
+/datum/heretic_knowledge/living_heart/on_research(mob/user)
+	. = ..()
+
+	var/obj/item/organ/heart/our_heart = user.getorganslot(ORGAN_SLOT_HEART)
+	if(our_heart)
+		our_heart.AddComponent(/datum/component/living_heart)
+
+/datum/heretic_knowledge/living_heart/on_lose(mob/user)
+	var/obj/item/organ/heart/our_heart = user.getorganslot(ORGAN_SLOT_HEART)
+	if(our_heart)
+		qdel(our_heart.GetComponent(/datum/component/living_heart))
+
+/datum/heretic_knowledge/living_heart/recipe_snowflake_check(mob/living/user, list/atoms, list/selected_atoms, turf/loc)
+	var/obj/item/organ/heart/our_heart = user.getorganslot(ORGAN_SLOT_HEART)
+	if(!our_heart || HAS_TRAIT(our_heart, TRAIT_LIVING_HEART))
+		return FALSE
+
+	if(our_heart.status == ORGAN_ORGANIC)
+		return TRUE
+
+	else
+		for(var/obj/item/organ/heart/nearby_heart in atoms)
+			if(nearby_heart.status == ORGAN_ORGANIC && nearby_heart.useable)
+				selected_atoms += nearby_heart
+				return TRUE
+
+		return FALSE
+
+
+/datum/heretic_knowledge/living_heart/on_finished_recipe(mob/living/user, list/selected_atoms, turf/loc)
+
+	var/obj/item/organ/heart/our_heart = user.getorganslot(ORGAN_SLOT_HEART)
+
+	if(our_heart.status != ORGAN_ORGANIC)
+		var/obj/item/organ/heart/our_replacement_heart = locate() in selected_atoms
+		if(our_replacement_heart)
+			user.visible_message("[user]'s [our_replacement_heart.name] bursts suddenly out of [user.p_their()] chest!")
+			INVOKE_ASYNC(user, /mob/proc/emote, "scream")
+			user.apply_damage(20, BRUTE, BODY_ZONE_CHEST)
+
+			our_replacement_heart.Insert(user, special = TRUE, drop_if_replaced = TRUE)
+			our_heart.throw_at(get_edge_target_turf(user, pick(GLOB.alldirs)), 2, 2)
+			our_heart = our_replacement_heart
+
+	if(!our_heart)
+		CRASH("[type] somehow made it to on_finished_recipe without a heart. What?")
+
+	if(our_heart in selected_atoms)
+		selected_atoms -= our_heart
+	our_heart.AddComponent(/datum/component/living_heart)
+	to_chat(user, span_warning("You feel your [our_heart.name] begin pulse faster and faster as it awakens!"))
+	playsound(user, 'sound/magic/demon_consume.ogg', 50, TRUE)
+	return TRUE
+
+/**
+ * Allows the heretic to craft a spell focus.
+ * They require a focus to cast advanced spells.
+ */
+/datum/heretic_knowledge/amber_focus
+	name = "Amber Focus"
+	desc = "Allows you to transmute a sheet of glass and a pair of eyes to create an Amber Focus. \
+		A focus must be worn in order to cast more advanced spells."
+	required_atoms = list(
+		/obj/item/organ/eyes = 1,
```

Comment:
eyes are an awfully specific item for the recipe of something that every heretic is going to need. "john just butchered a head and only took its eyes, he's a heretic" ("what a noob, doesn't he know to make flashlight eyes instead?") is gonna become the next "john printed a knife, he's a heretic" ("what a noob, doesn't he know to make a shiv instead?"), except way easier for your accuser to justify.